Makefile

```diff
@@ -7,6 +7,9 @@ INTERACTIVE := $(shell [ -t 0 ] && echo 1)
 
 # BUILD_DRY_RUN determines the value of the --dry-run flag of the build command. Should be 'true' or 'false'.
```

Comment:
```suggestion
# BUILD_DRY_RUN enables the build command's dry-run mode. In this mode, the build does not produce any permanent artifacts, e.g., an AMI. Should be 'true' or 'false'.
```

Reply:
It's not about the comment, it's that I wouldn't have even guessed that I would have to read the Makefile to understand this variable even exists. With this warning, it gives that hint.

Reply:
Good point. Would you like to move my text here to the warning you added?